NeosModLoader/AssemblyLoader.cs

```diff
@@ -85,7 +90,8 @@ internal static AssemblyFile[] LoadAssembliesFromDir(string dirName)
                 }
             }
 
-            return assemblyFiles.ToArray();
+            // remove duplicate assemblies and return the list
+            return assemblyFiles.Distinct(AssemblyFile.AssemblyComparer).ToArray();
```

Comment:
Well, my thought process here was that now that mod loading is recursive, someone could put a bunch of mods in a folder and call it a modpack.
If someone then puts two of those in at once, they want to load mods that are in both only once.
But yea, having that be logged probably makes more sense. (Which it will be by your deduplicator if I just take this deduplication out.)